docs/spec_draft.md

```diff
@@ -705,6 +706,57 @@ lexicographic comparison on the (real, imaginary) pairs.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.multiply
+
+### Semantics
+
+Performs element-wise product of two tensors `lhs` and `rhs` and produces a
+`result` tensor. For integer element types, if the element-wise product has an
+unsigned/signed overflow/underflow, the result is implementation-defined and one
```

Comment:
https://en.wikipedia.org/wiki/Arithmetic_underflow suggests that "the term underflow normally refers to floating point numbers only". Let's update the wording here, in `add` and in `subtract`.

Reply:
From https://en.wikipedia.org/wiki/Integer_overflow
```The term underflow is most commonly used for floating-point math and not for integer math. However, many references can be found to integer underflow. When the term integer underflow is used, it means the ideal result was closer to minus infinity than the output type's representable value closest to minus infinity. When the term integer underflow is used, the definition of overflow may include all types of overflows, or it may only include cases where the ideal result was closer to positive infinity than the output type's representable value closest to positive infinity.```

In that case should we just use the word overflow to mean both kinds of overflows.

Reply:
Yes, I think we should use just the word "overflow". Please also do the same for `add` and `subtract`.